lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -442,50 +470,62 @@ struct ConvertLayoutOpConversion
     // unsigned elems = getTotalElemsPerThread(srcTy);
     auto vals = getTypeConverter()->unpackLLElements(loc, adaptor.getSrc(),
                                                      rewriter, srcTy);
-    unsigned inVec = 0;
-    unsigned outVec = 0;
-    auto paddedRepShape = getScratchConfigForCvtLayout(op, inVec, outVec);
 
     unsigned outElems = getTotalElemsPerThread(dstTy);
     auto outOrd = getOrder(dstLayout);
     SmallVector<Value> outVals(outElems);
 
+    unsigned inVec = 0;
+    unsigned outVec = 0;
+    SmallVector<unsigned> repShape;
+    triton::gpu::SharedEncodingAttr sharedLayout;
+    if (srcLayout.isa<BlockedEncodingAttr>() &&
```

Comment:
Reorder conditions for readability. Possibly remove BlockedEncodingAttr from this condition